myuw/static/vue/components/accounts/upass.vue

```diff
@@ -0,0 +1,155 @@
+          <p>
+            Your U-PASS does not work during summer quarter unless you are registered
+            for a class or are a
+            <a href="https://facilities.uw.edu/transportation/employee-u-pass#8"
+              data-linklabel="TEMP Pass"
+            >
+              temporary employee
+            </a>.
+          </p>
+        </div>
+        <a :href="getUrl" data-linklabel="U-Pass">
+          U-PASS not working?
+        </a>
+      </div>
+      <div v-else id="upass-not-current">
+        <div v-if="!employee" id="upass-notices-for-students">
+          <div v-if="seattle">
+            <p>
+              If you are registered for a quarter, your U-PASS will work one week before
+              the quarter starts.
+            </p>
+            <div v-if="in_summer">
+              <h4>Summer U-PASS Use</h4>
+              <p>
+                Your U-PASS does not work during summer quarter unless you are registered
+                for a class or are a
+                <a
+                  href="https://facilities.uw.edu/transportation/employee-u-pass#8"
+                  data-linklabel="Temporary Employee U-PASS"
+                >
+                  temporary employee
+                </a>.
+              </p>
+            </div>
+          </div>
+          <div v-else id="upass-notices-for-non-sea-studs">
+            <p v-if="(bothell || tacoma)">
+              If you <a :href="getPurchaseUrl" data-linklabel="Purchase U-PASS">purchase</a>
+              a U-PASS for a quarter, your U-PASS will work one week before the quarter starts.
+            </p>
+          </div>
+        </div>
+        <ul>
+          <li>
+            <a :href="getWhatIsUrl" data-linklabel="What is U-PASS">
+              What is the U-PASS?
+            </a>
+          </li>
+          <li v-if="pce">
+            <a href="https://facilities.uw.edu/transportation/student-purchased-u-pass"
+                  data-linklabel="CC Student Purchase U-PASS"
+            >
+              Purchasing a U-PASS
+            </a>
+          </li>
+        </ul>
+      </div>
+    </template>
+  </uw-card>
+</template>
+<script>
+import {mapGetters, mapState, mapActions} from 'vuex';
+import Card from '../_templates/card.vue';
+export default {
+  components: {
+    'uw-card': Card,
+  },
+  computed: {
+    ...mapGetters('upass', {
+      isReady: 'isReady',
+      isErrored: 'isErrored',
+      statusCode: 'statusCode',
+    }),
+    ...mapState({
+      employee: (state) => state.user.affiliations.employee,
+      student: (state) => state.user.affiliations.student,
+      tacoma: (state) => state.user.affiliations.tacoma,
+      bothell: (state) => state.user.affiliations.bothell,
+      seattle: (state) => state.user.affiliations.seattle,
+      pce: (state) => state.user.affiliations.pce,
+      display_activation: (state) => state.upass.value.display_activation,
+      in_summer: (state) => state.upass.value.in_summer,
```

Comment:
eslint should have complained about this, we are doing camel case for our variable names

Reply:
Nope, we may want to let eslint check the camel  case.